app/main.py

```diff
@@ -1,8 +1,42 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int, brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: float, clean_power: int,
+                 average_rating: float, count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = round(average_rating, 1)
+        self.count_of_ratings = count_of_ratings
+
+    def calculate_washing_price(self, car: Car) -> float:
+        if car.clean_mark < self.clean_power:
+            price = (car.comfort_class * (
+                self.clean_power - car.clean_mark)
+                * self.average_rating / self.distance_from_city_center)
+            return round(price, 1)
+        return 0
+
+    def wash_single_car(self, car: Car) -> None:
+        if car.clean_mark < self.clean_power:
```

Comment:
This is a code quality issue. The condition `if car.clean_mark < self.clean_power:` is repeated in multiple methods. To avoid code duplication, you could create a separate method that checks if the car needs washing and use it in the `wash_single_car`, `calculate_washing_price` and `serve_cars` methods.